migration/003_add_deleted_at_column.sql

```diff
@@ -0,0 +1,7 @@
+-- +migrate Up
+-- SQL in section 'Up' is executed when this migration is applied
+ALTER TABLE "user" ADD COLUMN deleted_at Timestamp Without Time Zone;
```

Comment:
It may be dangerous to store the time without time zone.
If you don't need it - pls use UTC. Than any values that is stored in the DB will have the same time zone.

Reply:
Thank you. We took into account all the comments and made a new commit with them

Reply:
Done